src/Helper/Scripts.php

```diff
@@ -74,39 +72,36 @@ public function addCond($cond, $src, $pos = 100)
      *
      * @param mixed $script The script
      * @param int   $pos    The script position in the stack.
+     * @param array $attr The additional attributes
      *
      * @return Scripts
      *
      * @access public
      */
-    public function addInternal($script, $pos = 100)
+    public function addInternal($script, $pos = 100, array $attr = array())
     {
-        $attr = $this->escaper->attr(array(
-            'type' => 'text/javascript'
-        ));
+        $attr = $this->attr(null, $attr);
         $tag = "<script $attr>$script</script>";
         $this->addElement($pos, $tag);
         return $this;
     }
 
     /**
-     * addCondInternal
+     * Add Conditional internal script
      *
      * @param mixed $cond   The conditional expression for the script.
      * @param mixed $script The script
      * @param int   $pos    The script position in the stack.
+     * @param array $attr   The additional attributes
      *
-     * @return mixed
-     * @throws exceptionclass [description]
```

Comment:
What happened to this exceptionclass ? Wondering from where it is called though.